manifests/init.pp

```diff
@@ -11,7 +11,7 @@
 #   @see data/common.yaml
 #
 # @param dconf_hash
-#   Settings specific to dconf, Gnome 3, and MATE
```

Comment:
I think something like 'dconf settings specific to GNOME 3' would make more sense